tools/shoestring/tests/healthagents/test_websockets.py

```diff
@@ -22,10 +22,15 @@
 
 def test_should_run_for_api_role():
 	# Act + Assert:
-	assert should_run(NodeConfiguration(NodeFeatures.API, *([None] * 6)))
+	assert should_run(NodeConfiguration(NodeFeatures.API, None, None, None, None, True, None, None))
```

Comment:
Should there be a test for full api be False vs None?

Reply:
The `full_api` flag is boolean, so True and False should be fine.
But no harm in adding more tests.